### ISSUE TITLE
If I really want to use hoodie.js in production, what possible problem i will be faced in the future?

### ISSUE THREAD

**Issue by steve21124** [hoodiehq/hoodie#336]
sorry if I create a new issue for deploying hoodie in production (I aware that you guys said it is developer preview product). I love hoodie feature what kind of problem I might be facing if I deploy hoodie in production?

My plan is to deploy hoodie as part of mobile app in the front end and hoodie.js in backend. 

Thank you

**Comment by gr2m** [comment]
Thanks @steve21124, super glad to hear that you like Hoodie so far, despite it's early stage.

I've a hoodie based app in production myself now, and we have some more prototypes running for others. We described our own current deployment setup at https://github.com/hoodiehq/my-first-hoodie/blob/master/deployment.md. It's running smoothly, with a few thousand users per day. Once the migration to Hoodie is done I'll write up some experiences on it.

I'll close the issue if you don't mind, but feel free to keep asking farther questions
